package.json

```
@@ -57,7 +57,7 @@
       "properties": {
         "metals.serverVersion": {
           "type": "string",
-          "default": "0.5.2",
+          "default": "0.6.0",
```

```suggestion
          "default": "0.6.1",
```

It's already out on Maven 🎉